cmd/image-builder/main.go

```diff
@@ -246,6 +272,20 @@ operating systems like Fedora, CentOS and RHEL with easy customizations support.
 	buildCmd.Flags().String("cache", "/var/cache/image-builder/store", `osbuild directory to cache intermediate build artifacts"`)
 	rootCmd.AddCommand(buildCmd)
 
+	// XXX: add --format=json too?
+	describeImgCmd := &cobra.Command{
+		Use:          "describe-image <distro> <image-type>",
```

Comment:
```suggestion
		Use:          "describe-image <image-type>",
```
Distro is an optional arg, right?

Reply:
Thanks, fixed now.